resources/user-agents/apps/site-monitors/site-con.json

```diff
@@ -21,9 +21,7 @@
         {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) SiteCon/8.10*",
           "properties": {
```

Comment:
Why is there a space between a star and a parenthesis ?

Reply:
This forces the space to be there inan UA to match the pattern. The UAs we found at the time we defined this rule had this space.
Do you have an UA without this space?

Reply:
I don't know. It's a theoretical question. The star includes the spaces, so we absolutely need to have a space in the pattern.

Reply:
As @mimmi20 mentioned, it does require the space to be there, which can be useful for only capturing valid UAs, it also increases the length of the pattern, which is important for priority.

I try to keep spaces where they belong in the patterns, but there are a few exceptions, especially in the Safari/Firefox/Chrome patterns which other browsers are based on.  I try to keep those patterns as short as possible, so patterns that are very similar, but only differ by a few characters (like `OPR` for Opera) have higher priority, and won't be mis-detected as Chrome, for example.